### PR TITLE
Musl fixes

### PR DESCRIPTION
**Pull request by andred**
These patches fix compilation against the musl C library.

### CODE REVIEW

**Comment by jforissier** [comment]
Hi @andred,

Thanks for the patches, look fine to me.

`Reviewed-by: Jerome Forissier <jerome.forissier@linaro.org>`

File: ta/crypt/arith_taf.c [hunk 1]
```diff
@@ -2,6 +2,7 @@
 /* Copyright (c) 2018, Linaro Limited */
 
 #include <arith_taf.h>
+#include <inttypes.h>
```
Comment (jenswi-linaro) [inline ta/crypt/arith_taf.c]: I don't expect that this file is compiled with musl-libc.
I agree with the change, it's just that the commit message isn't accurate for this file.

Comment (andred) [reply ta/crypt/arith_taf.c]: You're right. I've updated the subjects of all related patches

**Comment by jenswi-linaro** [comment]
`Acked-by: Jens Wiklander <jens.wiklander@linaro.org>`

**Comment by jforissier** [comment]
@andred would you please add @jenswi-linaro's `Acked-by:` tag to each commit? Then I can merge this PR.

**Comment by jforissier** [comment]
Thanks!

**Pull request merged**